source/cppexpose/include/cppexpose/reflection/Object.h

```diff
@@ -55,6 +55,9 @@ class CPPEXPOSE_API Object : public AbstractProperty
     */
     Object(const std::string & name);
 
+    Object(const Object &) = delete;
```

Comment:
Maybe if everything else would all be well documented I would be willing to argue about this. But since I have to demand comments over and over again, please just DO IT.